plugins/qiskit_executor/routes.py

```diff
@@ -301,11 +300,16 @@ def render(self, db_id, data: Mapping, errors: dict, valid: bool):
             msg = f"Could not load task data with id {db_id} to read parameters!"
             raise KeyError(msg)
 
-        params = CircuitParameterSchema().loads(db_task.data["parameters"])
+        params: CircuitParameters = CircuitParameterSchema().loads(
```

Comment:
Use `load` if paramaters are no longer serialized to a string (see side note in comment before)